packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -633,6 +634,21 @@ class DragTarget<T extends Object> extends StatefulWidget {
   ///
   /// The builder can build different widgets depending on what is being dragged
   /// into this drag target.
+  ///
+  /// [onWillAccept] or [onWillAcceptWithDetails] is called when a draggable
+  /// enters the target. If either return true, the data will appear in
+  /// `candidateData`, if they return false, the data will appear in
+  /// `rejectedData`.
+  ///
+  /// Typically the builder will check `candidateData` and `rejectedData` and
+  /// and build a widget that indicates the result of dropping the
+  /// `candidateData` onto this target.
+  ///
+  /// The `candidateData` and `rejectedData` are List types to support multiple
+  /// simultaneous drags.
+  ///
+  /// If you see unexpected `null` values in `candidateData` or `rejectedData`,
+  /// ensure that the `data` argument of the [Draggable] is not `null`.
```

Comment:
To [avoid saying "you"](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#use-the-passive-voice-recommend-do-not-require-never-say-things-are-simple):

"If unexpected `null` values appear in `candidateData` or `rejectData`,..."